app/core/speaker_diarization.py

```
@@ -203,20 +255,65 @@ def diarize_audio(self, audio_data: bytes) -> List[Dict]:
-        return {
-            "success": False,
-            "error": str(e),
-            "speaker_count": 0,        # ✅ 추가
-            "total_duration": 0.0,     # ✅ 추가
-            "segments": [],            # ✅ 추가
-            "timestamp": datetime.now().isoformat(),
-            "processing_method": "pyannote"  # ✅ 추가
-        }
+            return []
+
+    def map_speakers_to_mentor_mentee(self, segments: List[Dict], mentor_idx: int, mentee_idx: int) -> List[Dict]:
```

Ruff (0.12.2)

Unused method argument: `mentor_idx`

(ARG002)